src/page-modules/assistant/trip-pattern/index.tsx

```diff
@@ -0,0 +1,93 @@
+import { formatLocaleTime, secondsToDuration } from '@atb/utils/date';
+import { type TripPattern } from '@atb/page-modules/assistant/server/journey-planner/validators';
+import style from './trip-pattern.module.css';
+import { ComponentText, useTranslation } from '@atb/translations';
+import { Typo } from '@atb/components/typography';
+import { motion } from 'framer-motion';
+import { Button } from '@atb/components/button';
+import { MonoIcon } from '@atb/components/icon';
+import { TransportIconWithLabel } from '@atb/components/transport-mode/transport-icon';
+
+type TripPatternProps = {
+  tripPattern: TripPattern;
+  index: number;
+};
+export function TripPattern({ tripPattern, index }: TripPatternProps) {
+  const { t, language } = useTranslation();
+
+  const duration = secondsToDuration(tripPattern.duration, language);
+  const fromPlace = tripPattern.legs[0].fromPlace.name ?? '';
```

Comment:
Can this contain no legs? So this would cause fromPlace of undefined error

Reply:
That might very well be. Adding a `?.` to avoid that at least. Should probably be handled and not showing a trip pattern at all if that's the case? Have you experienced any trip patterns with no legs, @mortennordseth?

Reply:
Not that I can think of 🤔

Reply:
Added optional chaining to avoid error. Should look into handling no legs if that's ever the case.